src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int SWITCH_BOUND = 5;
+    private static final int RADIUS_OF_DEFAULT_FIGURE = 10;
+    private static final int ORIGIN = 1;
+    private static final int MAX_VALUE = 100;
+    private static final String COLOR_OF_DEFAULT_FIGURE = "white";
+
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
```

Comment:
Don't begin method implementation with an empty line. Remove the empty line at the start of the 'getRandomFigure()' method.